CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ endif(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 
 # Options
 
-option(ENABLE_TESTS "Enable all tests and checks" OFF)
```

Comment:
@sunweaver: I thing we should revert this change - it is inconsistent with everything else in Ayatana. In all our sources we provide defaults for quick and easy builds - testing is optional (mostly for packagers and developers in mind).